packages/grid/src/GridMetricCalculator.ts

```diff
@@ -53,27 +54,6 @@ export interface GridMetricState {
   draggingColumn: DraggingColumn | null;
 }
 
-/**
- * Retrieve a value from a map. If the value is not found and no default value is provided, throw.
- * Use when the value _must_ be present
- * @param map The map to get the value from
- * @param key The key to fetch the value for
- * @param defaultValue A default value to set if the key is not present
- * @returns The value set for that key
- */
-export function getOrThrow<K, V>(
```

Comment:
Instead of just getting rid of this (which would be a breaking API change for anybody using our `@deephaven/grid` package), just re-export the `getOrThrow` function, e.g.
```
export { getOrThrow } from '@deephaven/utils';
```
Otherwise we need to mark this change as breaking.